frontend/src/metabase/xray/components/InsightCard.jsx

```diff
@@ -175,17 +175,21 @@ export class VariationTrendInsight extends Component {
 
   render() {
     const { mode } = this.props;
+    const MODE_ADVERB_STRINGS = {
+      big: t`bigly`,
```

Comment:
Sorry I should have made it clear those were placeholder words, I didn't know the actual words, but I checked and they are the following:

```javascript
 const MODE_ADVERB_STRINGS = {
  increasing: t`increasingly`,
  decreasing: t`decreasingly`
}
```

I'd also move it above `export class VariationTrendInsight extends Component {` and rename it `VARIATION_MODE_ADVERB_STRINGS`

Thanks